addon/styles/_frost-tab.scss

```diff
@@ -1,9 +1,8 @@
-$frost-tab-active-border: $frost-color-blue-1;
+$frost-tab-active-border: $frost-color-grey-2;
```

Comment:
I thought this was only certain circumstances - we never have a blue for selection?